src/list-grid.scss

```diff
@@ -0,0 +1,232 @@
+@import './new-settings';
+@import './mixins';
+
+/*!
+.fd-grid-list
+*/
+$block: #{$fd-namespace}-grid-list;
+
+.#{$block} {
+  $fd-grid-list-border-radius: var(--sapElement_BorderCornerRadius);
+  $fd-grid-list-active-text-color: var(--sapList_Active_TextColor);
+  $fd-grid-list-item-box-shadow: var(--sapContent_Shadow0);
+  $fd-grid-list-mode-box-shadow: 0 0 0 0.125rem var(--sapList_SelectionBorderColor);
+  $fd-grid-list-highlight: (
+    'success': ('backgroundColor': var(--sapSuccessBorderColor)),
+    'error': ('backgroundColor': var(--sapErrorBorderColor)),
+    'warning': ('backgroundColor': var(--sapWarningBorderColor)),
+    'neutral': ('backgroundColor': var(--sapNeutralBorderColor))
+  );
+
+  @include fd-reset();
+
+  display: flex;
+  flex-direction: column;
+
+  .fd-list {
+    &__item {
```

Comment:
You are not supposed to modify the properties of the List directly here. You can create a new class and rely on the css specificity to overwrite the properties of the list-item. For example:
`div class="fd-list__item fd-grid-list__item"....</div>`

Then in `.fd-grid-list__item` you overwrite the properties coming from `fd-list__item`. This will also eliminate the need of using !important, which is not recommended.

Reply:
already not use `fd-list`